web/modules/custom/joinup_event/config/install/core.entity_view_display.node.event.default.yml

```diff
@@ -34,7 +34,6 @@ dependencies:
     - image
     - joinup_core
     - link
-    - options
```

Comment:
Comment is not related to this line of code, I use it for a general comment. This requirement is missing from the display of the event before-content areas:
```
Author (author photo, name and family name) a tile seprartor '|' and the publication date (gg/mm/aaaa). (for this refer also to ISAICP-3671)
```

Reply:
Ok, I added creation date and author.

Reply:
Thanks